com.trollworks.gcs/src/com/trollworks/gcs/preferences/ReferenceLookupPreferences.java

```diff
@@ -84,7 +112,43 @@ public ReferenceLookupPreferences(PreferencesWindow owner) {
             preferredSize.height = 200;
         }
         scroller.setPreferredSize(preferredSize);
-        add(scroller);
+        add(scroller, new PrecisionLayoutData().setGrabHorizontalSpace(true).setHorizontalSpan(2)
```

Comment:
Please keep this all on one line, like most of the rest of the code does.

Reply:
I think you're right. I stole the whole list of interfaces from another tab and meant to go back and remove it and didn't. Will do.

Reply:
Formatter fixed this.